the-collab-lab/tcl-24-smart-shopping-list
# Message for duplicated items on the Form added

## Description

This PR adds isItemDuplicated function in the Form component to validate if the item that the user tries to submit is already in the list and displays an error message when it is already in the list. 

## Related Issue

Closes #7

## Acceptance Criteria


## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

### After

![Screen Shot 2021-04-21 at 4 27 52 PM](https://user-images.githubusercontent.com/39011391/115623305-0326d400-a2c7-11eb-8475-bd2625ad73ad.png)

## Testing Steps / QA Criteria

- From your terminal, pull down the branch
- Run npm install to install the dependencies
- Run npm start to view the features in the browser
- Click AddItem button
- Fill in the form with an item that is already on the list
- Click on submit or send buttom.
- Check the alert "Item is already on the list"
